balancer-js/examples/createPool.ts

```diff
@@ -0,0 +1,96 @@
+import dotenv from 'dotenv';
+import { Wallet } from '@ethersproject/wallet';
+import { InfuraProvider } from '@ethersproject/providers';
+import { BalancerSDK, BalancerSdkConfig, Network, SeedToken, SwapType } from '../src/index';
+import { DAI } from './constants';
+
+dotenv.config();
+
+const { TRADER_KEY } = process.env;
+
+export interface tokens {
+  [name: string]: {
+    address: string;
+    decimals: number;
+  };
+}
+const WEIGHTED_POOL_FACTORY = '0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9';
```

Comment:
Could we add the pool factories to the `contracts.module.ts`? That way a user wouldn't have to worry about finding addresses.

Reply:
Fixed in commit: 2dc14d8d84c8bc2ebd811d6f0bf7480cc9fcdf45